jekyll/_cci2/hello-world.md

```diff
@@ -7,11 +7,30 @@ categories: [getting-started]
 order: 4
 ---
 
-This document describes how to configure your Linux, Android or macOS project to run on CircleCI 2.0 after you [sign up]({{ site.baseurl }}/2.0/first-steps/).  
+This document describes how to configure your Linux, Android or macOS project to run on CircleCI 2.x after you [sign up]({{ site.baseurl }}/2.0/first-steps/).  
```

Comment:
Does the 2.0 first steps link include information about 2.1?

Reply:
thanks, I will update it to say 2.x instead of 2.0

Reply:
ok, updated